services-geojson/src/main/java/com/mapbox/geojson/Feature.java

```diff
@@ -229,13 +264,14 @@ public BoundingBox bbox() {
    *
    * @return a String containing this features unique identification or null if one wasn't given
    *   during creation.
-   * @since 1.0.0
+   * @since 5.0.0
    */
   @Nullable
-  public String id() {
+  public Object id() {
```

Comment:
would be great to include as idAsString and idAsNumber facility methods  (if the type doesn't match throw an exception).